CRM/Contact/Import/Parser/Contact.php

```diff
@@ -482,25 +485,6 @@ public function import($onDuplicate, &$values) {
     //now we create new contact in update/fill mode also.
     $contactID = NULL;
     if ($createNewContact || ($this->_updateWithId)) {
-      // @todo - there are multiple places where formatting is done that need consolidation.
```

Comment:
This was added in conjuction with `CRM_Contact_Import_Parser_ContactTest::testPreferredLanguageImport` as a generic way to handle preferred_language labels. The same is already being achieved in the `getTransformedFieldValue` so this is now obsolete